pkg/paho-mqtt/Makefile

```
@@ -0,0 +1,10 @@
+PKG_NAME = paho-mqtt
+PKG_URL = https://github.com/eclipse/paho.mqtt.embedded-c.git
+PKG_VERSION = 29ab2aa29c5e47794284376d7f8386cfd54c3eed
+PKG_LICENSE = EDL
+
+include $(RIOTBASE)/pkg/pkg.mk
+
+all:
+	"$(MAKE)" -C $(PKG_BUILDDIR)/MQTTPacket/src/ -f $(CURDIR)/Makefile.$(PKG_NAME)-packet
+	"$(MAKE)" -C $(PKG_BUILDDIR)/MQTTClient-C/src/ -f $(CURDIR)/Makefile.$(PKG_NAME)
```

#14289 was merged so this Makefile must be adapted:

```suggestion
	"$(MAKE)" -C $(PKG_SOURCE_DIR)/MQTTPacket/src/ -f $(CURDIR)/Makefile.$(PKG_NAME)-packet
	"$(MAKE)" -C $(PKG_SOURCE_DIR)/MQTTClient-C/src/ -f $(CURDIR)/Makefile.$(PKG_NAME)
```